clippy_lints/src/indexing_slicing.rs

```diff
@@ -17,19 +17,20 @@ declare_clippy_lint! {
     /// ### Why is this bad?
     /// This will always panic at runtime.
     ///
-    /// ### Known problems
-    /// Hopefully none.
-    ///
     /// ### Example
-    /// ```no_run
+    /// ```rust,no_run
```

Comment:
Just a note: `rust` is implicit in rustdoc and also added by the metadata collector for the lint list. I don't think it's wrong to add it, but any implementations should not rely on `rust` as a language indicator being present, IMO.

This can remain, there is no benefit in removing it :upside_down_face:

Reply:
Yeah, I just add it when I see it missing. I like being explicit with these things :smile:.